### PR TITLE
Renaming conformance definition proposals to best practice proposals

### PR DESCRIPTION
**Pull request by taylor**
Goal: Move from the conformance definition language to best practices for the proposals

Renaming CCDP (CNF Conformance Definition Proposal) to CBPP (CNF Best Practice Proposal)

- [x] update docs to use best practice vs conformance everywhere related to CNF WG activities 
- [x] update use cases
- [x] update templates
- [x] update directory name
- [x] update links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # Cloud Native Network Function Working Group (CNF-WG)
 
 
-The Cloud Native Network Function WG (CNF WG) operates under the aegis of CNCF. The focus of the CNF WG is to define the process around evaluating the cloud nativeness of networking applications, aka CNFs. We collaborate with the [CNF Conformance test suite project](https://github.com/cncf/cnf-conformance/blob/master/README-testsuite.md) who work on the mechanics of the conformance tests.
+The Cloud Native Network Function WG (CNF WG) operates under the aegis of CNCF. The focus of the CNF WG is to define the process around evaluating the cloud nativeness of networking applications, aka CNFs. We collaborate with the [CNF Test Suite project](https://github.com/cncf/cnf-conformance/blob/master/README-testsuite.md) who work on the mechanics of the conformance tests.
 
-The goal for the group is to create a software conformance program that any network application implementation can use to determine how well it follows cloud native principles and best practices.
+The primary goal for this group is to provide a set of Cloud native + Kubenative best pratices for network applications.
```
Comment (ildikov) [inline README.md]:
```suggestion
The primary goal for this group is to provide a set of cloud native + Kubenative best practices for network applications. 
```

File: cbpps/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Quick start for the CBPP process
+
+Socialize an idea with the CNF WG. You can send your idea to the mailing list, or add it to the agenda for one of the upcoming meetings. Make sure that others think the work is worth taking up and will help review the CBPP and any code changes required.
+
+Follow the process outlined in the [CBPP process document](0001-cnf-best-practice-proposal-process.md) and use the [CBPP template](NNNN-cbpp-template.md) to create a new proposal. Submit a pull request to add the proposal to the repo.
+
+## CNF Use Cases
+
+CBPP shall always refer to at least one of CNF WG use cases. Those use cases serve as the practical and concrete basis for deriving, discussing and evaluating the best practices for CNFs. Their value is in providing context and sanity check for CBPPs. With help of use cases we making sure that CBPPs are addressing specific problem or need from the real world. Please use [Use Case Template](../use-case/NNNN-UC-template.md) to contribute an use case.
```
Comment (xmulligan) [inline cbpps/README.md]:
```suggestion
CBPP shall always refer to at least one of CNF WG use cases. Those use cases serve as the practical and concrete basis for deriving, discussing and evaluating the best practices for CNFs. Their value is in providing context and reality check for CBPPs. With help of use cases we making sure that CBPPs are addressing specific problem or need from the real world. Please use [Use Case Template](../use-case/NNNN-UC-template.md) to contribute an use case.
```

**Review by xmulligan — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
 
-The goal for the group is to create a software conformance program that any network application implementation can use to determine how well it follows cloud native principles and best practices.
+The primary goal for this group is to provide a set of cloud native + Kubenative best practices for network applications. 
 
 The [CNF WG Charter](charter.md) futher outlines the scope of our group activities as well as intended deliverables.
 
-The CNF Conformance program will evaluate cloud nativeness based on the set of best practices in which the CNF WG has selected following the [CNF Conformance Definition Proposal](ccdps). Anyone in the CNF WG is welcome to participate in the process from creation to selection of proposals.
-
+The [CNF Test Suite](https://github.com/cncf/cnf-conformance) will support testing a set of these best practices to allow developers and network operators to evaluate how well a network application follows cloud native principles and best practices. Proposals which have been adopted by the CNF WG are listed in the [CNF Best Practice Proposal](cbpp) folder.
```
Comment (xmulligan) [inline README.md]:
```suggestion
The [CNF Test Suite](https://github.com/cncf/cnf-conformance) will support testing a set of these best practices to allow developers and network operators to evaluate how well a network application follows cloud native principles and best practices. Proposals which have been adopted by the CNF WG are listed in the [CNF Best Practice Proposal](/cbpp) folder.
```

**Comment by taylor** [comment]
@xmulligan  +1
@jeffsaelens +1
@fkautz +1

**Pull request merged**